build/root/WORKSPACE

```diff
@@ -73,23 +73,23 @@ docker_pull(
     digest = "sha256:86176bc8ccdc4d8ea7fbf6ba4b57fcefc2cb61ff7413114630940474ff9bf751",
```

Comment:
the digests are what actually matter here.

Reply:
oops, thanks. Done.